WordPress/gutenberg
# show comment form if there is no comment on any block

Part of - https://github.com/WordPress/gutenberg/issues/66377



## What? & What?

Focus the reply field when clicking on the toolbar button, it currently doesn’t do anything when the sidebar is already open.

## How?

The sidebar will now show a comment form for any selected block that does not yet have comments. This encourages user interaction and feedback.


## Testing Instructions

1. Enable experimental inline commenting option under Gutenberg -> Experiments.
2. Edit any post/page.
3. Open the sidebar and click on any block or add any new block to see comment form in the sidebar.

## Screenshots or screencast 
https://www.loom.com/share/c2c3032a3f8443629514a57ac9668487?sid=452256dd-f06c-450b-b465-e7d3a4ceed86
